scanpy/preprocessing/_highly_variable_genes.py

```diff
@@ -193,6 +197,26 @@ def _highly_variable_genes_seurat_v3(
         return df
 
 
+@numba.njit(cache=True)
+def _clip_sparse(
```

Comment:
Minor point, is the name `_clip_sparse` final..? not sure what else to call it tho, fast clip sum square anything... feel free to ignore this point if its just me, is concise enough to be clear anyways

Reply:
```suggestion
def _sum_and_sum_squares_clipped(
```

?